.github/scripts/smoke-test.sh

```diff
@@ -29,12 +29,23 @@ oc create secret docker-registry pull-secret \
  --docker-password=$RED_HAT_PASSWORD \
  --docker-email=$RED_HAT_EMAIL
 
-oc apply -f ${WORKDIR}/operator-rbac.yaml
+cat <<EOF > ${WORKDIR}/service-account.yaml
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: hazelcast-enterprise-operator
+  labels:
+    app.kubernetes.io/name: hazelcast-enterprise-operator
+    app.kubernetes.io/instance: hazelcast-enterprise-operator
+    app.kubernetes.io/managed-by: hazelcast-enterprise-operator
+EOF
+
+oc apply -f ${WORKDIR}/service-account.yaml
```

Comment:
Nit

```suggestion
cat <<EOF | kubectl apply -f -
apiVersion: v1
kind: ServiceAccount
metadata:
  name: hazelcast-enterprise-operator
  labels:
    app.kubernetes.io/name: hazelcast-enterprise-operator
    app.kubernetes.io/instance: hazelcast-enterprise-operator
    app.kubernetes.io/managed-by: hazelcast-enterprise-operator
EOF
```

Reply:
Looks a lot better, I will change it. Thanks :)